modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsSnapshotV2.java

```diff
@@ -826,6 +838,48 @@ public CacheMetricsSnapshotV2(CacheMetrics loc, Collection<CacheMetrics> metrics
         return keySize;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public long getCacheTouches() {
+        return cacheTouches;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public long getCacheTouchHits() {
+        return cacheTouchHits;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public long getCacheTouchMisses() {
+        return cacheTouchMisses;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public float getCacheTouchHitPercentage() {
+        long hits = cacheTouchHits;
+        long touches = cacheTouches;
+
+        if (touches == 0)
+            return 0;
+
+        return (float) hits / touches * 100.0f;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public float getCacheTouchMissPercentage() {
+        long misses = cacheTouchMisses;
+        long touches = cacheTouches;
+
+        if (touches == 0)
+            return 0;
+
+        return (float) misses / touches * 100.0f;
+    }
+
```

Comment:
Please update `writeExternalData` and `readExternalData` as well in order to properly serialize instances of this class.